storage/gcsemu/gcsemu.go

```diff
@@ -152,7 +168,7 @@ func (g *GcsEmu) Handler(w http.ResponseWriter, r *http.Request) {
 			g.gapiError(w, http.StatusBadRequest, fmt.Sprintf("unsupported value for alt param to PATCH: %q\n%s", alt, maybeNotImplementedErrorMsg))
 		}
 	case "POST":
-		if bucket == "" && r.URL.Query().Get("alt") == "json" {
```

Comment:
I guess this is over specified?  The Java client doesn't send this extra query param?